### PR TITLE
chore: Add Speedial Playground

### PR DESCRIPTION
**Pull request by jugshaurya**
Issue: #107
Speed dial Playground added with sidebar being updated with speeddial link. @pranshuchittora please review

Screenshot from 2021-04-21 14-19-35

### CODE REVIEW

File: src/content/SpeedDial/speeddial.playground.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  });
+
+  return (
+    <React.Fragment>
+      <Playground params={params} />
+    </React.Fragment>
+  );
+};
+
+export default SpeedDialPlayground;
```
Comment (arpitBhalla) [inline src/content/SpeedDial/speeddial.playground.jsx:67]: Can you increase height, so that actions become visible

Comment (jugshaurya) [reply src/content/SpeedDial/speeddial.playground.jsx:67]: Sure. will do today

**Comment by jugshaurya** [comment]
@flyingcircle, please review

**Comment by flyingcircle** [comment]
merge conflict

**Comment by jugshaurya** [comment]
@flyingcircle , resolved conflicts

**Pull request merged**